### DISCUSSION TITLE
Clarification on support for features in development

### DISCUSSION THREAD

**Discussion by sfc-gh-dschonberg** [Snowflake-Labs/terraform-provider-snowflake#1751] [Announcements]
Hello all,

We sometimes get requests to support features in preview mode. We would like to clarify that our criteria for adding support for a feature is that it be generally available with documentation. We believe that holding to this standard will ensure stability for our customers. By making this post, we hope to provide predictability for how we address issues with the provider.

Thank you for your support!
Dan Schonberg on behalf of the Snowflake Terraform Provider team.

**Comment by sfc-gh-pkommini** [comment]
That makes sense. GA seems like a good time to **_release_** features . :thumbsup:. And also when there's no docs it's hard to support that feature in the provider.

However, we have a lot of folks who are prototyping and testing PuPr features and have Snowflake Data infrastructure that they need to quickly spin up and quickly tear down and Terraform would greatly help in that rapid prototyping phase of those features.

Would it be possible to have a `pupr` prefix or suffix convention for versions where we can contribute and test PuPr features? That way folks who are prototyping those features have an option to use the terraform provider for those features.
